### ISSUE TITLE
Franz handles hardware media keys

### ISSUE THREAD

**Issue by JakubRimal** [meetfranz/franz#1798]
**Describe the bug**
Franz sometimes handles the hardware media keys (e.g. play key from the user's keyboard) which is very annoying because the user is not able to control his player.

**To Reproduce**
1. Start your media player application (e.g. Rhythmbox) and play some music.
1. Pause music.
2. Start Franz
3. Make a call in MS Teams.
1. End a call.
4. Press play media key.
5. Some MS Teams sound (ringing) plays.

**Expected behavior**
Play media key controls the user's media player.

**Desktop (please complete the following information):**
 - OS: Linux Mint 19.3 Cinnamon
 - Franz version: "5.4.0"

**Additional context**
It is possible to disable hardware media key handling in Chromium-like browsers:

1. Visit chrome://flags
1. Set Hardware Media Key Handling to Disabled

I think that Franz should have hardware media key handling disabled by default and/or configurable in settings.

**Comment by M0UL** [comment]
i have the same problem with whatsapp videos claiming the media keys.

**Comment by HumusFoodz** [comment]
Yes same issue here (OS: Linux Mint Cinnamon 18.2, Franz: 5.4.0). It seems to hijack the controls the same way chromium does by default.
The only workaround for now to get the media controls to function for media player is to quit franz leaving the media player open and restart which makes the media controls direct into the player again.

**Comment by tchesa** [comment]
Same problem here.

**Comment by SpikeVanHellsing** [comment]
Still an issue. I like to watch youtube while doing other things and as it turns out that franz seems to take priority of my media keys. is there any way to stop this?
